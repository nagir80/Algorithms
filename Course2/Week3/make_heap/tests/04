10
5 10 8 9 4 1 6 7 2 3 
