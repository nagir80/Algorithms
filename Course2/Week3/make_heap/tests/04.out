6
4 9
3 7
2 5
1 4
4 9
0 2
